spring-pulsar/src/main/java/org/springframework/pulsar/core/Resolved.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.core;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+
+import org.springframework.lang.Nullable;
+
+public final class Resolved<T> {
```

Comment:
Do you mind adding javadoc w/ `@author`?

Reply:
Sure